pkg/liqoctl/status/utils/resources/resources_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019-2022 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	sharev1alpha1 "github.com/liqotech/liqo/apis/sharing/v1alpha1"
+	"github.com/liqotech/liqo/pkg/client/clientset/versioned/scheme"
+	"github.com/liqotech/liqo/pkg/consts"
+	"github.com/liqotech/liqo/pkg/liqoctl/status/utils/resources"
+)
+
+var _ = Describe("Shared Resources utility functions", func() {
+
+	var (
+		clientBuilder fake.ClientBuilder
+		ctx           context.Context
+		cl            client.Client
+		resExpected   corev1.ResourceList
+		cpuQuantity   = resource.NewScaledQuantity(1000, resource.Kilo)
+		memQuantity   = resource.NewQuantity(2000, resource.BinarySI)
+		podsQuantity  = resource.NewQuantity(1, resource.DecimalSI)
+		otherQuantity = resource.NewQuantity(4000, resource.DecimalSI)
```

Comment:
It would be better to initialize these in the BeforeEach block, to avoid issues if they get mutated.